conda_forge_tick/migrators.xsh

```diff
@@ -724,7 +744,7 @@ class Pinning(Migrator):
         upd = "\n".join(lines) + "\n"
         with open(os.path.join(recipe_dir, "meta.yaml"), "w") as f:
             f.write(upd)
-        Rebuild.bump_build_number(os.path.join(recipe_dir, "meta.yaml"))
+        self.set_build_number(os.path.join(recipe_dir, "meta.yaml"))
```

Comment:
I think this will have the wrong behavior. This will set the build number to 1000 for the rebuilds, when it needs to be set to the current build number + 1000

Reply:
Oh you are right, this has an extra `cls.bump_number` in it.  @mariusvniekerk - would you mind putting in a fix please?

Reply:
Err that should not set it to 1000, the set to 1000 bit only applies to the Version migrator